e2e_playwright/widget_state.py

```diff
@@ -16,6 +16,17 @@
 
 import streamlit as st
 
+# we use session_state to store the callback function value
+# instead of using st.write directly in the button callback,
+# because we have observed that the button-click is sometimes not
+# dispatched correctly when the rerun from the textArea's onBlur event
+# is too fast which leads to a changing number of elements per rerun (because the
+# callback's write would disappear briefly).
+if "btn_callback" not in st.session_state:
+    st.session_state.btn_callback = "Input: "
+
+st.write(st.session_state.btn_callback)
```

Comment:
Probably works fine for the test since its only a single rerun, but for multi-rerun we probably need to reset the `st.session_state.btn_callback` to the default value after every write